plugin/gatsby-browser.js

```diff
@@ -40,5 +41,14 @@ export const onInitialClientRender = (_, options) => {
     }
   };
 
-  frame();
+  if (season) {
```

Comment:
We could do `if(season && new Date() ...)` not sure if you're a fan of nested `if` statements or not?

Reply:
No real preference.